projects/go-lib/src/lib/go-config.service.ts

```diff
@@ -1,89 +1,62 @@
 import { Injectable } from '@angular/core';
 import { BehaviorSubject } from 'rxjs';
-import { GoConfigInterface } from './go-config.model';
-
-interface RGB {
-  r: number;
-  g: number;
-  b: number;
-}
+import {
+  BrandingMode,
+  GoConfigInterface,
+  LogoConfig,
+  ThemeColors
+} from './go-config.model';
 
 @Injectable()
 export class GoConfigService {
-  config: BehaviorSubject<GoConfigInterface> = new BehaviorSubject<GoConfigInterface> ({
-    brandColor: '#65B360',
-    headerBrandingEnabled: false
-  });
+  private config: GoConfigInterface = {
+    brandColor: ThemeColors.brand,
+    brandingMode: BrandingMode.company,
+    logoConfig: {
+      logo: '',
+      logoLink: ''
+    }
```

Comment:
Should this be initialized without the `logoConfig` here? In the html for the header we have an `*ngIf="logoConfig"` and then we assume that if we have the logoConfig then we will have a logoLink, which won't be the case when this is initialized.

Reply:
Sure, that shouldn't matter to remove this. :+1: